docs/designers-developers/glossary.md

```diff
@@ -1,21 +1,61 @@
+
+<dt>Block name</dt>
+<dd>A unique identifier for a block type, consisting of a plugin-specific namespace and a short label describing the block's intent. e.g. `core/image`</dd>
+
+<dt>Block type</dt>
+<dd>In contrast with the blocks composing a particular post, a block type describes the blueprint by which any block of that type should behave. So while there may be many images within a post, each behaves consistent with a unified image block type definition.</dd>
+
+<dt>Classic block</dt>
+<dd>A block which embeds the TinyMCE editor as a block, TinyMCE was the base of the previous core editor. Older content created prior to the block editor will be loaded in to a single Classic block.</dd>
+
+<dt>Dynamic block</dt>
+<dd>A type of block where the content of which may change and cannot be determined at the time of saving a post, instead calculated any time the post is shown on the front of a site. These blocks may save fallback content or no content at all in their JavaScript implementation, instead deferring to a PHP block implementation for runtime rendering.</dd>
+
+<dt>Inspector</dt>
+<dd>A block settings region shown in place of the post settings when a block is selected. Fields may be shown here to allow the user to customize the selected block.</dd>
+
+<dt>Post settings</dt>
+<dd>A sidebar region containing metadata fields for the post, including scheduling, visibility, terms, and featured image.</dd>
+
+<dt>RichText</dt>
+<dd>A common component enabling rich content editing including bold, italics, hyperlinks, etc. It is not too much unlike the single editor region of the legacy post editor, and is in fact powered by the same TinyMCE library.</dd>
+
+<dt>Reusable block</dt>
+<dd>A block that is saved and then can be shared as a reusable, repeatable piece of content.</dd>
+
+<dt>Sidebar</dt>
+<dd>The panel on the right which contains the document and block settings. The sidebar is toggled using the Settings gear icon.</dd>
+
+<dt>Serialization</dt>
+<dd>The process of converting a block's attributes object into HTML markup, typically occurring when saving the post.</dd>
+
+<dt>Static block</dt>
+<dd>A type of block where the content of which is known at the time of saving a post. A static block will be saved with HTML markup directly in post content.</dd>
+
+<dt>TinyMCE</dt>
+<dd><a href="https://www.tinymce.com/">TinyMCE</a> is a web-based JavaScript WYSIWYG (What You See Is What You Get) editor.</dd>
+
+<dt>Toolbar</dt>
+<dd>A set of button controls. In the context of a block, usually referring to the toolbar of block controls shown above the selected block.</dd>
+
+<dt>Template</dt>
+<dd>A block template is defined as a list of block items. Such blocks can have predefined attributes, placeholder content, and be static or dynamic. Block templates allow to specify a default initial state for an editor session. See <a href="developers/block-api/block-templates.md">Templates documentation</a> for more.</dd>
```

Comment:
URLs have to be fully relative to the root of the repo and back up. So it would be…

```
../../../developers/block-api/block-templates.md
```

Reply:
Also I'd expand this a bit because templates can also be used in the context of InnerBlocks, and they're not just lists — they can be a full nested structure. Something like this might be good:

> A template is a pre-defined arrangement of blocks, possibly with predefined attributes or placeholder content. You can provide a template for a post type, to give users a starting point when creating a new piece of content, or inside a custom block with the `InnerBlocks` component. See the templates documentation for more information.

Reply:
👍 Updated